docs/tutorial/6_visualization.md

```diff
@@ -59,84 +58,171 @@ This takes a list which defines which axes to plot as the slice and which to use
+If instead we want to plot the image formed by the raster scan at a particular wavelength and Stokes value, we can change which axes we specify as the x and y axes of the plot.
+For this example we will slice the data down to a particular region of interest.
 
-```{warning}
-Plotting a raster scan of VISP data is currently very slow due to known performance issues in how varying pointing over the raster is handled. See issue [#256](https://github.com/DKISTDC/dkist/issues/256) for more details.
+```{code-cell} python
+small_ds = ds[0, 150:300, :, 1010:-1010]
 ```
 
-```{code-cell} ipython3
-ds.plot(plot_axes=[None, 'y', None, 'x'])
+Note that since this removes the first axis, the `plot_axes` argument is now only three items long.
+
+```{code-cell} ipython
+# Plot 'raster scan step number' on the y axis and 'spatial along slit' on the x axis
+small_ds.plot(plot_axes=['y', None, 'x'])
 plt.show()
 ```
 
 You may have noticed this plot took somewhat longer to draw than the previous one.
 This is for the same reason as we discussed when talking about reducing the size of dataset downloads: when you slice across the array in a different direction to how it's stored in the files, you have to reference multiple files to create the slice.
-So while the first plot only had to load values from one file at a time, the one above needs to get one line of the array from each of 1000 files in order to draw the slice.
+So while the first plot only had to load values from one file at a time, the one above needs to get one line of the array from each of 150 files in order to draw the slice.
 If you try to animate it, it then needs to do this again at every step.
 
+```{warning}
+For the above example we have very deliberately sliced the data to a size that is manageably small.
+Due to some known issues in the plotting code, plotting slices across the data in this way is extremely slow and is not really viable for a full-size dataset.
+See issue [#256](https://github.com/DKISTDC/dkist/issues/256) for more details.
+```
+
 You can also use `plot_axes` to create a line plot, by specifying only one axis of the data.
-So to plot a spectrum at a fixed Stokes, time and raster location we can tell plot to use the dispersion axis as the x axis.
+So to plot a spectrum at a fixed Stokes, time and raster location we can tell `plot` to use the dispersion axis as the x axis.
 
-```{code-cell} ipython3
+```{code-cell} ipython
 ds.plot(plot_axes=[None, None, 'x', None])
 plt.show()
 ```
 
 It is also possible to slice the data manually and just plot the result.
 This of course creates a new dataset so it will only plot the axes that remain, without sliders or the ability to step through the values of the other axes.
 
-```{code-cell} ipython3
-:tags: [skip-execution]
-
-ds[0, :, 400, :].plot()
+```{code-cell} ipython
+# Plot the same data as above
+ds[0, 0, :, 0].plot()
 plt.show()
 ```
 
-## More advanced plotting
+### Using AIA as a Context Image
+
+#### Fetching an AIA Image
```

Comment:
This feels kinda out of place, also it duplicates what I added here: https://github.com/DKISTDC/dkist/pull/492/files#diff-f5104acda48fa6b068e8705f2a8b81af1e7666df7b70dc6b0628889ab9ff1e42R65-R89